tfx/components/example_validator/executor_test.py

```diff
@@ -46,26 +46,45 @@ def testDo(self):
         self._testMethodName)
 
     validation_output = standard_artifacts.ExampleAnomalies()
-    validation_output.uri = os.path.join(output_data_dir, 'output')
+    validation_output.uri = output_data_dir
```

Comment:
let's just use os.path.join(output_data_dir, 'output'), thus if we have other output in the future, it's not conflict